src/components/layouts/Header.js

```diff
@@ -0,0 +1,56 @@
+import { PRIVATE_MENU_ITEMS, PUBLIC_MENU_ITEMS } from "../../constants/menu";
+import { isLoggedIn, onLogout } from "../../stores/auth";
+import { removeBasePath } from "../../utils/routePath";
+
+const Header = () => {
+  const init = () => {
+    const $nav = document.getElementById("nav");
+    if ($nav) {
+      $nav.addEventListener("click", (e) => {
+        e.preventDefault();
+
+        if (e.target.id === "logout") {
+          onLogout();
+        } else if (e.target.href && e.target.href !== location.href) {
+          window.navigate(e.target.href.replace(location.origin, ""));
+        }
+      });
+    }
+  };
+
+  const getClass = (href) => {
+    if (location.hash) {
+      return location.hash === `#${href}`
+        ? "text-blue-600 font-bold"
+        : "text-gray-600";
+    }
+
+    return removeBasePath(location.pathname) === href
+      ? "text-blue-600 font-bold"
+      : "text-gray-600";
+  };
```

Comment:
```suggestion
  const getClass = (href) => {
    const isActive = location.hash === `#${href}` || removeBasePath(location.pathname) === href;
    return isActive ? "text-blue-600 font-bold" : "text-gray-600";
  };
```
리턴 값이 같아서 합쳐보았습니다 ㅎ_ㅎ